src/solver/infeasible-problem-analysis/constraint-slack-analysis.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "unfeasibility-analysis.h"
+
+namespace Antares::Optimization
+{
+
+class ConstraintSlackAnalysis : public UnfeasibilityAnalysis
+{
+public:
+    ConstraintSlackAnalysis() = default;
+    void run(operations_research::MPSolver* problem) override;
+    void printReport() override;
```

Comment:
could be const

Reply:
Sorry I'm confused : what could be **const**